### DISCUSSION TITLE
Contribution guidance

### DISCUSSION THREAD

**Discussion by dotdot0** [vlang/v#17705] [General]
How I can start contributing to vlang as a beginner??
Some simple issues I can start with?

**Comment by JalonSolov** [comment]
I would start with https://github.com/vlang/v/issues/7047

Take any of the unchecked items, read through the code, and try to write useful documentation for them, following these guidelines:  https://github.com/vlang/v/blob/master/doc/docs.md#writing-documentation

**Comment by ttytm** [comment]
I'm not part of the V-Team or anything, below is just a general approach that I think is working: 

A very good beginning is imho to simply start to program in V a lot. If coming across any issues, try narrowing them down to a minimal reproducible example (MRE). This will help you to isolate the problem and create a valuable report. The reproduction might even be useful as a testcase. Additionally, based on the isolated issue and if you think a fix is feasible, explore the codebase and try to actually fix it. If it's not in reach yet, take a look at how the issues are resolved and learn from reading the code.
